pyluba/aliyun/cloud_gateway.py

```diff
@@ -520,12 +520,15 @@ def send_cloud_command(self, command: bytes):
         )
 
         # TODO move to using  InvokeThingServiceRequest()
+
+        message_id = str(uuid.uuid4())
+
         body = IoTApiRequest(
-            id=str(uuid.uuid4()),
+            id=message_id,
```

Comment:
if message id isn't used in multiple places it should remain as `str(uuid.uuid4())`